pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/main/java/org/apache/pinot/plugin/stream/kinesis/KinesisPartitionGroupOffset.java

```diff
@@ -40,10 +40,20 @@
 public class KinesisPartitionGroupOffset implements StreamPartitionMsgOffset {
   private final String _shardId;
   private final String _sequenceNumber;
+  private final OffsetStartStatus _startStatus;
```

Comment:
Seems it is not persistent to the offset string. Is this intentional?
We need to think about how to migrate the current table to this new format

Reply:
This was a miss. Made the change. 
I have ensured backward compatibility so for old table we don't need to worry. the start status would always be RESUME for them which is the current behavior as well.